random-reviewer-picker/main.js

```diff
@@ -0,0 +1,142 @@
+const github = require("@actions/github");
+const core = require("@actions/core");
+const { random } = require("lodash");
+
+/**
+ * Fetch and transform inputs into usable variables
+ **/
+function getInputs() {
+  const reviewer_list = core
+    .getInput("reviewer_list")
+    .split(",")
+    .map(i => i.replace(" ", ""));
+  const reviewer_amount = parseInt(core.getInput("reviewer_amount"));
+  const maintainer_list = core
+    .getInput("maintainer_list")
+    .split(",")
+    .map(i => i.replace(" ", ""));
+  const maintainer_amount = parseInt(core.getInput("maintainer_amount"));
+  const skip_busy = !!core.getInput("skip_busy");
+
+  const GITHUB_TOKEN = core.getInput("GITHUB_TOKEN");
+
+  return {
+    reviewer_list,
+    reviewer_amount,
+    maintainer_list,
+    maintainer_amount,
+    skip_busy,
+    GITHUB_TOKEN
+  };
+}
+
+/**
+ * Get user "busy" status from github API
+ **/
+async function isUserBusy(octokit, userHandle) {
+  core.info(`Checking status of "${userHandle}"`);
+
+  const { user } = await octokit.graphql({
+    query: `
+    query userStatus($user: String!) {
+      user(login: $user) {
+        status {
+          indicatesLimitedAvailability
+        }
+      }
+    }`,
+    user: userHandle
+  });
+
+  // If user never changed their status, then status is null
+  if (user.status && user.status.indicatesLimitedAvailability) {
+    return userStatus.user.status.indicatesLimitedAvailability;
+  }
+
+  return false;
```

Comment:
How about using -

```js
return (user.status && user.status.indicatesLimitedAvailability || false);
```

Reply:
I kinda like it how it is, a bit more verbose and easy to change if needed

Reply:
Sure, works for me too then :)